ext/pcntl/pcntl.c

```diff
@@ -692,8 +692,21 @@ PHP_FUNCTION(pcntl_signal_get_handler)
 		Z_PARAM_LONG(signo)
 	ZEND_PARSE_PARAMETERS_END();
 
-	if (signo < 1 || signo > 32) {
-		zend_argument_value_error(1, "must be between 1 and 32");
+#if !defined(__APPLE__)
+	static int sigmax = NSIG;
```

Comment:
From https://www.gnu.org/software/libc/manual/html_node/Standard-Signals.html:
>    The value of this symbolic constant is the total number of signals defined. Since the signal numbers are allocated consecutively, NSIG is also one greater than the largest defined signal number. 

So shouldn't this _also_ be `NSIG-1`, since it's one larger than the largest signal number, or am I misunderstanding something ?

Reply:
Yes, I forgot to update up there.